[2021-06-27 07:16:30,807] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T07:16:24.179692+00:00 [queued]>
[2021-06-27 07:16:30,877] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T07:16:24.179692+00:00 [queued]>
[2021-06-27 07:16:30,878] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:16:30,879] {taskinstance.py:1068} INFO - Starting attempt 1 of 2
[2021-06-27 07:16:30,882] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 07:16:30,913] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): export_postgresql_customer> on 2021-06-27T07:16:24.179692+00:00
[2021-06-27 07:16:30,933] {standard_task_runner.py:52} INFO - Started process 5081 to run task
[2021-06-27 07:16:30,965] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgretomysql', 'export_postgresql_customer', '2021-06-27T07:16:24.179692+00:00', '--job-id', '44', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/postgresql-collect.py', '--cfg-path', '/tmp/tmpg9mnsolk', '--error-file', '/tmp/tmpwq1jm6sf']
[2021-06-27 07:16:30,969] {standard_task_runner.py:77} INFO - Job 44: Subtask export_postgresql_customer
[2021-06-27 07:16:31,122] {logging_mixin.py:104} INFO - Running <TaskInstance: postgretomysql.export_postgresql_customer 2021-06-27T07:16:24.179692+00:00 [running]> on host b69deda3a989
[2021-06-27 07:16:31,246] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=tech@innospark.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgretomysql
AIRFLOW_CTX_TASK_ID=export_postgresql_customer
AIRFLOW_CTX_EXECUTION_DATE=2021-06-27T07:16:24.179692+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-27T07:16:24.179692+00:00
[2021-06-27 07:16:31,248] {logging_mixin.py:104} INFO - Entering export_postgresql_to_csv SELECT * FROM test.customer
[2021-06-27 07:16:31,275] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:16:51,658] {base.py:78} INFO - Using connection to: id: docker-postgres. Host: 192.168.1.27, Port: 5432, Schema: , Login: postgres, Password: ***, extra: {}
[2021-06-27 07:16:51,682] {python.py:151} INFO - Done. Returned value was: None
[2021-06-27 07:16:51,713] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=postgretomysql, task_id=export_postgresql_customer, execution_date=20210627T071624, start_date=20210627T071630, end_date=20210627T071651
[2021-06-27 07:16:51,811] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-06-27 07:16:51,887] {local_task_job.py:151} INFO - Task exited with return code 0
